ui/app/src/Views/SilenceList/SilenceView.elm

```diff
@@ -18,34 +18,31 @@ import Views.SilenceForm.Types exposing (SilenceFormMsg(NewSilenceFromMatchers))
 
 view : Silence -> Html Msg
 view silence =
-    div [ class "d-inline-flex align-items-center justify-content-start w-100" ]
-        [ datesView silence.startsAt silence.endsAt
-        , div [ class "" ] (List.map matcherButton silence.matchers)
-        , div [ class "ml-auto d-inline-flex align-self-stretch p-2", style [ ( "border-left", "1px solid #ccc" ) ] ]
-            [ editButton silence
-            , deleteButton silence
-            , detailsButton silence.id
-            ]
-        ]
+    li [ class "align-items-start list-group-item border-0 alert-list-item p-0 mb-4" ]
+        [ div [ class "w-100 mb-2 d-flex align-items-start" ]
+            [ case silence.status.state of
+                Active ->
+                    dateView "Ends" silence.endsAt
 
+                Pending ->
+                    dateView "Starts" silence.startsAt
 
-datesView : Time -> Time -> Html Msg
-datesView start end =
-    i [ class "d-inline-flex align-items-center", style [ ( "border-right", "1px solid #ccc" ) ] ]
-        [ dateView start
-        , i [ class "text-muted" ] [ text "-" ]
-        , dateView end
+                Expired ->
+                    dateView "Expired" silence.endsAt
+            , detailsButton silence.id
+            , editButton silence
+            , deleteButton silence
+            ]
+        , div [ class "" ] (List.map matcherButton silence.matchers)
         ]
 
 
-dateView : Time -> Html Msg
-dateView time =
-    i
-        [ class "h-100 p-2 d-flex flex-column justify-content-center, text-muted"
-        , style [ ( "font-family", "monospace" ) ]
+dateView : String -> Time -> Html Msg
+dateView string time =
+    span
+        [ class "text-muted align-self-center mr-2"
         ]
-        [ span [] [ text <| Utils.Date.timeFormat time ]
-        , small [] [ text <| Utils.Date.dateFormat time ]
+        [ text (string ++ " " ++ Utils.Date.timeFormat time ++ ", " ++ Utils.Date.dateFormat time)
```

Comment:
Do we need to display the Silence state here? Isn't it already obvious by the new tabs in the Silence view?